x-pack/plugin/searchable-snapshots/qa/rest/src/test/java/org/elasticsearch/xpack/searchablesnapshots/rest/FsSearchableSnapshotsIT.java

```diff
@@ -19,14 +18,8 @@ protected String repositoryType() {
 
     @Override
     protected Settings repositorySettings() {
-        final Settings.Builder settings = Settings.builder();
-        settings.put("location", System.getProperty("tests.path.repo"));
-        if (randomBoolean()) {
-            settings.put("compress", randomBoolean());
-        }
-        if (randomBoolean()) {
-            settings.put("chunk_size", randomIntBetween(100, 1000), ByteSizeUnit.BYTES);
```

Comment:
I think we should keep this, maybe `rarely()`, or else we'll never test any of the code paths for multi-chunk files.

Reply:
SearchableSnapshotsIntegTests uses a `fs` repository with a `chunk_size` setting:

https://github.com/elastic/elasticsearch/blob/95070652c5e6e987ba1d6d550d19d9d50aeb3743/x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java#L84

I think it's better to have it there and not in this QA test, what do you think?